kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/ObjectCreator.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.good.patterns.food2door;
+
+public class ObjectCreator
```

Comment:
w jakim celu dodałeś te klasę? Żeby pobawić się tworzeniem obiektów w taki sposób? 
Zwracasz tutaj Object co będzie powodowało problemy w pozostałych częściach kodu, gdyż będziesz musiał wykonywać rzutowanie.

Reply:
No cóż. Wczoraj na pierwszy rzut Twojego oka wyglądało to bardziej optymistycznie:) Generalnie miałem do wyboru zostawić kod w innej klasie lub go wydzielić, co bardziej odpowiada SOLID, oczywiście nie musiałem od razu tworzyć kreatora obiektów tylko kreator implementacji VendorCheckout. Miałem obiekcje, co do zbytniej ogólności tego rozwiązania, ale z rozwiązań niepożądanych kojarzę jedynie tworzenie zmiennych typu Object (zbyt ogólnych) i przypisywanie im implementacji konkretnych klas. Nie wiem w czym to może szkodzić, ale nie jestem i nie byłem przekonany do tego rozwiązania.

Reply:
Zmieniłem tak, by niesławne rzutowanie było w klasie tworzącej obiekt.